src/main/java/net/neoforged/neoforge/event/entity/living/ProjectileWeaponLoadEvent.java

```diff
@@ -23,7 +23,7 @@ public abstract class ProjectileWeaponLoadEvent extends LivingEvent {
     private final ItemStack weaponItem;
     private final ItemStack projectileItem;
 
-    public ProjectileWeaponLoadEvent(LivingEntity entity, ItemStack weaponItem, ItemStack projectile) {
+    ProjectileWeaponLoadEvent(LivingEntity entity, ItemStack weaponItem, ItemStack projectile) {
```

Comment:
I would say just make the constructor public and add the annotation `@ApiStatus.Internal`